src/utils/safelyInvoke.spec.js

```diff
@@ -0,0 +1,17 @@
+import safelyInvoke from './safelyInvoke';
+
+describe('safelyInvoke', () => {
+  test('should return the original function', () => {
+    const myFunction = jest.fn();
+    expect(safelyInvoke(myFunction)).toBe(myFunction);
```

Comment:
Let's call the function returned by safelyInvoke and assert that its called.
So something like this.

```
const myFunction = jest.fn();
const safeFunction = safelyInvoke(myFunction);
safeFunction();

expect(myFunction).toHaveBeenCalled();
```